exporter/awss3exporter/documentation.md

```
@@ -0,0 +1,47 @@
+[comment]: <> (Code generated by mdatagen. DO NOT EDIT.)
```

markdownlint-cli2 (0.22.0)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "comment"

(MD053, link-image-reference-definitions)